generative_model_training_all_labels/01_generative_model_dag_all_labels.ipynb

```diff
@@ -0,0 +1,1734 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    l2_param_grid = np.linspace(0.01, 5, num=5)
</pre><p>I found it was better to use logspace than linspace when doing hyperparameter optimization since random samples tend to be large. This may not be an issue though since you're using a grid. Also probably not a big enough difference to be worth rerunning if it would take very long</p><p> </p><p>For reference,</p><p><code>np.linspace(0.01, 5, num=5)</code> returns array([0.01&nbsp;, 1.2575, 2.505 , 3.7525, 5.&nbsp;&nbsp;])</p><p>while</p><p><code>np.logspace(-2, 0, num=5)</code> returns [0.01&nbsp;&nbsp;&nbsp;, 0.03162278, 0.1&nbsp;&nbsp;&nbsp;&nbsp;, 0.31622777, 1.&nbsp;&nbsp;&nbsp;&nbsp;]</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/snorkeling-full-text/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>